e2e/multiserver_multiagent_test.go

```diff
@@ -0,0 +1,106 @@
+package e2e
+
+import (
+	"context"
+	"strconv"
+	"testing"
+	"time"
+
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/klient/wait/conditions"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+func TestMultiServer_MultiAgent_StaticCount(t *testing.T) {
+	serverServiceHost := "konnectivity-server.kube-system.svc.cluster.local"
+	agentServiceHost := "konnectivity-agent.kube-system.svc.cluster.local"
+	adminPort := 8093
+	replicas := 3
+
+	serverStatefulSetCfg := StatefulSetConfig{
+		Replicas: 3,
```

Comment:
nit: use `replicas` instead of `3` here and below in the agent config

Reply:
Done!